jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/main/JmixUserIndicator.java

```diff
@@ -110,6 +120,11 @@ public void refreshUser() {
         getContent().removeAll();
 
         UserDetails user = currentUserSubstitution.getAuthenticatedUser();
```

Comment:
Technically, we want to set a user from `currentUserSubstitution` only if we can't obtain a fresh one from `userRepository`, so this code can be moved to the `catch`

Reply:
We load user from `userRepository` by `userName` which we get from `currentUserSubstitution.getAuthenticatedUser()`